pmm/main.go

```diff
@@ -143,11 +143,11 @@ Use 'pmm-admin config' to define server user and password.`, a.Config.ServerAddr
 	}
 
 	// Check Consul status.
-	if leader, err := a.consulAPI.Status().Leader(); err != nil || leader != "127.0.0.1:8300" {
```

Comment:
@rnovikovP @askomorokhov @percona-csalguero so you are fine with this change? It will also fix #22